index.js

```diff
@@ -1,39 +1,86 @@
-const express = require('express')
-const app = express()
-const port = 3001
-
-// app.get('/', (req, res) => res.send('Steam reviews service?'))
+const db = require('./db/connection.js');
+const express = require('express');
+const app = express();
+const port = 3001;
 
 app.listen(port, () => console.log(`Steam reviews service. listening at http://localhost:${port}`))
 app.use(express.static('./client/dist'));
 
-// SAMPLE API CALLS
 
-app.get('/api/reviewcount/all/summary/:productId', ( req, res) => {
-    res.send( sampleAllSummary );
-});
+// tool: return summary text from %
+var rating = per => 
+[ { min: 95, text:'Overhwelmingly Positive' },
```

Comment:
Typo: Overwhelmingly. A linter would help keep these nasty typos at bay!

Reply:
Does a linter actually fix spelling errors for non-coding words? Wow! Hopefully it'll be worth all the pain it was getting it going the first time!

Reply:
There's definitely a plugin for eslint for spellcheck at least